src/stretchy.js

```diff
@@ -180,9 +180,11 @@ const svgSpan = function(
         let viewBoxWidth = 400000;  // default
         const label = group.value.label.substr(1);
         if (utils.contains(["widehat", "widetilde", "utilde"], label)) {
+            // $FlowFixMe: implied by the labels above.
+            const grp: ParseNode<"accent"> | ParseNode<"accentUnder"> = group;
```

Comment:
Should this also include `ParseNode<"xArrow">`?

Reply:
No, since that doesn't have `grp.value.base` which I access below. `"widehat", "widetilde", "utilde"` (from the `if` condition) all correspond to one of these two types, which is what I mean by "implied by the labels above".

Reply:
I see.  Can you add those details to the `$FlowFixMe` comment?

Reply:
Done. Please see if it's clear enough.